src/bind.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Bind charm business logic."""
+
+import logging
+
+from charms.operator_libs_linux.v2 import snap
+
+import constants
+import exceptions
+
+logger = logging.getLogger(__name__)
+
+
+class BindService:
+    """Bind service class."""
+
+    def reload(self) -> None:
```

Comment:
It looks like the method is unused, is it intended to be used later ? If not, then let's remove it

Reply:
We will need a way to reload the service when the configuration will change